ui/components/app/collectibles-items/collectibles-items.js

```diff
@@ -202,7 +202,11 @@ export default function CollectiblesItems({
                   key={`collectible-${i}`}
                   className="collectibles-items__item-wrapper"
                 >
-                  <Card padding={0} justifyContent={JUSTIFY_CONTENT.CENTER}>
+                  <Card
+                    padding={0}
+                    justifyContent={JUSTIFY_CONTENT.CENTER}
+                    className="collectibles-items__item-wrapper__card"
+                  >
```

Comment:
Adding overflow hidden to the card so the image being contained doesn't overlap the border.